manifests/bucketeer/charts/batch/values.yaml

```diff
@@ -0,0 +1,72 @@
+image:
+  repository: ghcr.io/bucketeer-io/bucketeer-batch
+  pullPolicy: IfNotPresent
+
+fullnameOverride: "batch-server"
+
+namespace: default
+
+env:
+  project:
+  mysqlUser:
+  mysqlPass:
+  mysqlHost:
+  mysqlPort:
+  mysqlDBName:
+  notificationService: localhost:9001
+  environmentService: localhost:9001
+  autoOpsService: localhost:9001
+  experimentService: localhost:9001
+  eventCounterService: localhost:9001
+  featureService: localhost:9001
+  webURL:
+  logLevel: info
+  port: 9090
+  metricsPort: 9002
+  timezone: UTC
+  refreshInterval:
+
+affinity: {}
+
+nodeSelector: {}
+
+replicaCount: 1
+
+envoy:
+  image:
+    repository: envoyproxy/envoy-alpine
```

Comment:
Please update the envoy repository and tag version.
https://github.com/bucketeer-io/bucketeer/blob/main/manifests/bucketeer/charts/account-apikey-cacher/values.yaml#L53-L54